src/renderer/l10n/ko.js

```diff
@@ -0,0 +1,266 @@
+  repeatYourPassphrase: '여기에 패스워드를 반복해 주십시오',
+  save: '저장',
+  saveAndBackupTheKeys: '아래의 키 저장 및 백업.',
+  savedWallet: '저장된 지갑',
+  saving: '저장 중...',
+  search: '검색',
+  secondsAgo: '{seconds} 바로 전',
+  selectAWallet: '지갑 선택',
+  selectIco: 'ICO 선택',
+  sell: '팔다',
+  send: '전송',
+  sendGasClaim: '가스 요청 전송.',
+  sendingClaim: '~{amount} 가스 요청을 전송합니다.',
+  sentClaim: '~{amount} 가스 요청을 전송합니다.',
+  setMarketRelayed: 'setMarket 호출 중계',
+  settings: '설정',
+  size: '크기',
+  spread: '스프레드',
+  status: '상태',
+  steps: '단계:',
+  submit: '제출',
+  submittingMultipleTimesWarning: '여러 번 제출하면 자금이 손실 될 수 있습니다.;',
+  success: '성공!',
+  systemFee: '요금 제도',
+  testDEXVersion: '현재 Aphelion DEX 테스트 버전을 보고있습니다. Mainnet이 곧 공개될 예정입니다.',
+  theDexContractHasBeenUpdated: `DEX 계약이 업데이트되었습니다. DEX를 계속 사용하려면 해당
+   지갑 업그레이드가 필요합니다. 아래의 컨트럴 버튼을 클릭하여 주문을 취소하고 지갑에서
+    계약금을 인출 한 후 최신 버전을 다운로드하십시오.`,
+  theFeeForCompletingYourTradeWillBe: '귀하의 거래를 완료하는 수수료는 {amount} APH입니다',
+  theFeeForCompletingYourTradeWillDepend: `귀하의 거래를 완료하는 데 드는 비용은 일치하는
+   최종 제안 수에 따라 다르지만 {min} APH와 {max} APH 사이에서 결정됩니다`,
+  thisIsAPreliminaryDemo: `현재 Aphelion DEX의 예비 데모 및 테스트 버전을 사용하고 계십니다.
+   Neo Testnet에서 작동하므로 성능이 저하 될 수 있습니다. Mainnet이 곧 공개될 예정입니다.`,
+  thisMeansThatItIsIneligible: '게시 전용 주문에 부적격합니다.',
+  thisOrderRequires: `본 주문을 완성하려면 {quantity} {symbol}이 필요합니다.
+  현재 계약 잔액은 {balance} {symbol}입니다.
+  우선 {deposit} {symbol}을 제출하고 주문서를 보내면 성공적으로 처리할 수 있습니다.`,
+  thisOrderWouldTakeTheFollowing: '본 주문은 아래의 사항이 필요합니다:',
+  thisWillBeAMakerOrder: `본 주문은 제조자의 주문으로 누군가가 채택할 때까지 남습니다. 
+본 거래에 대한 수수로는 청구되지 않습니다.`,
+  thisWillBeAMakerOrderLeftOnTheBook: `본 주문은 제조자의 주문으로 누군가가 채택할 때까지 남습니다. 
+본 거래에 대한 수수로는 청구되지 않습니다.`,
+  time: '시간',
+  to: '까지',
+  toDownload: '다운로드',
+  token: '토큰',
+  tokenSaleSuccessful: '토큰 판매 참여 성공. 현재 {symbol} 잔액이 {balance}입니다.',
+  tokenStats: '토큰 통계',
+  totalSupply: '총 공급',
+  tradeDEX: 'DEX 거래',
+  tradeHistory: '거래 내역',
+  tradesWillBePoweredBy: `거래는 APH 토큰으로 가동되며 Aphelion Distributed Exchange
+   Asset Ledger (D.E.A.L) 스마트 계약을 통해 모든 APH 토큰 보유자에게 재배포됩니다.`,
+  transaction: '거래',
+  transferAllNEO: '모든 NEO를 본인에게 전송.',
+  transferredNEO: '{amount} NEO가 귀하에게 전송되었습니다.',
+  transferringNEO: '{amount} NEO가 귀하에게 전송 중 입니다.',
+  twentyFourHour: '24시간',
+  twentyFourHourChange: '24시간 변경',
+  type: '유형',
+  unableToReachNetwork: '네트워크에 연결할 수 없습니다.',
+  unableToReachTradingServer: '거래 서버에 연결할 수 없습니다.',
+  unconfirmed: '확인되지 않음',
+  unitsTotal: '단위 합계',
+  useAnIcoScriptHash: 'ICO 스크립트 해시 사용',
+  value: '가치',
+  vol: '용량',
+  volume: '용량',
+  waitForGasClaim: 'GAS 청구 확인을 위해 기다려주십시오.',
+  waitForGasClaimTransaction: 'GAS 청구 거래 세부 정보를 위해 기다려주십시오.',
+  waitForNEO: 'NEO 전송 확인을 위해 기다려주십시오.',
+  waitingForConfirmation: '확인을 기다리는 중...',
+  waitingForGasClaim: 'GAS 청구 확인을 기다리는 중입니다.',
+  waitingForNEO: 'NEO 전송 확인을 기다리는 중입니다.',
+  waitingForTransaction: '거래가 블록 탐색기에 표시될때까지 기다리는 중...',
+  walletBalance: '지갑 잔액',
+  walletBalanceContractBalance: '지갑 잔액: {walletBalance}\n계약 잔액: {contractBalance}',
```

Comment:
This key should also show open orders balance now; see the english one; since we don't have the text in korean for "Open Orders" try google translate for it for now.